recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -1,3 +1,45 @@
 APP ?= chatbot
+PORT ?= 8501
 
 include ../../common/Makefile.common
+
+CHROMEDRIVER_EXISTS ?= $(shell command -v chromedriver)
```

Comment:
Should this all be in Common?

Reply:
If there are other recipes that use `chromedriver` then most definetly. If there are not it is considered specific to  this recipe. I couldnt find any other references to `chromedriver` so I stuck this here.

Reply:
As per your and Michaels comments I have abstracted the chromedriver instllation into common. Now only each recipe will provide the relative installation path to the recipes dir for installation.